pymc_marketing/mmm/delayed_saturated_mmm.py

```diff
@@ -142,6 +144,102 @@ def _save_input_params(self, idata) -> None:
         idata.attrs["validate_data"] = json.dumps(self.validate_data)
         idata.attrs["yearly_seasonality"] = json.dumps(self.yearly_seasonality)
 
+    def _create_likelihood_distribution(
+        self,
+        dist: Dict,
+        mu: SharedVariable,
+        observed: Union[np.ndarray, pd.Series],
+        dims: str,
+    ) -> TensorVariable:
+        """
+        Create and return a likelihood distribution for the model.
+
+        This method prepares the distribution and its parameters as specified in the
+        configuration dictionary, validates them, and constructs the likelihood
+        distribution using PyMC.
+
+        Parameters
+        ----------
+        dist : Dict
+            A configuration dictionary that must contain a 'dist' key with the name of
+            the distribution and a 'kwargs' key with parameters for the distribution.
+        observed : Union[np.ndarray, pd.Series]
+            The observed data to which the likelihood distribution will be fitted.
+        dims : str
+            The dimensions of the data.
+
+        Returns
+        -------
+        TensorVariable
+            The likelihood distribution constructed with PyMC.
+
+        Raises
+        ------
+        ValueError
+            If 'kwargs' key is missing in `dist`, or the parameter configuration does
+            not contain 'dist' and 'kwargs' keys, or if 'mu' is present in the nested
+            'kwargs'
+        """
+        allowed_distributions = [
```

Comment:
LogNormal as well?

Reply:
I actually avoided it because I'm not entirely sure about using `LogNormal` on the likelihood, which would require changing the `mu` - it's a bit complicated. Maybe we should hold off on it for now and tackle it later? Otherwise, we'll have to create new tests and handle more exceptions in the code when the user selects this type.

Reply:
Why does the `LogNormal` require changing mu? Don't listen to @nialloulton :D

Reply:
Hehe okay then... Let's make it happen, I'll add it during the day.

Reply:
Reminder, still missing?